src/crepe/api/Button.h

```diff
@@ -43,6 +40,9 @@ class Button : public UIObject {
 	friend class InputSystem;
 	//! Indicates whether the mouse is currently hovering over the button
 	bool hover = false;
+
+	//! variable indicating if transform is relative to camera(false) or world(true)
+	bool world_space = false;
```

Comment:
Goed dat je `bool world_space` in een struct hebt gezet. Maar je moet dan nog wel overal even `button.world_space` vervangen door `button.data.world_space`.